fasjsonclient/clientbuilder/__init__.py

```diff
@@ -0,0 +1,20 @@
+from fasjsonclient.openapi_client import Configuration, ApiClient, DefaultApi
+from fasjsonclient.openapi_client.rest import RESTClientObject
+
+from . import patch
+
+
+class ClientBuilder(object):
+  """Class that builder a fasjson openapi rest client object"""
```

Comment:
Typo: class that *builds*

Reply:
I forgot to mention I am the king of mistyping :-)